src/Acuminator/Acuminator.Vsix/ServiceLocation/DelegatingServiceLocator.cs

```diff
@@ -1,90 +1,90 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using CommonServiceLocator;
+﻿//using System;
```

Comment:
Please delete file as it is not used

Reply:
I’m not sure - it can serve as an example for service locator. But we also have got history. @dropsonic what do you think?

Reply:
@SENya1990 ok, let's remove it and postpone the investigation until we really need DI.

Reply:
Ok, then I'll remove both files

Reply:
Done